<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphDialog</class>
 <widget class="QDialog" name="GraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphs</string>
  </property>
  <widget class="QCustomPlot" name="scorePlot" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="timePlot" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="depthPlot" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="npsPlot" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1001</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
